code/datums/goon_mutations_disabilities.dm

```diff
@@ -0,0 +1,185 @@
+/datum/mutation/human/wacky
+	name = "Wacky"
+	quality = MINOR_NEGATIVE
+	text_indication = "<span class='sans'>You feel an off sensation in your voicebox.</span>"
+	lose_indication = "<span class='notice'>The off sensation passes.</span>"
+
+/datum/mutation/human/wacky/say_mod(var/message)
+	if(message)
+		message = "<span class='sans'>[message]</span>"
+	return message
+
+/datum/mutation/human/mute
+	name = "Mute"
+	quality = NEGATIVE
+	text_indication = "<span class='notice'>You feel unable to express yourself at all.</span>"
+	lose_indication = "<span class='notice'>You feel able to speak freely again.</span>"
+
+/datum/mutation/human/mute/on_acquiring(mob/living/carbon/human/owner)
+	if(..())	return
+	owner.disabilities |= MUTE
+
+/datum/mutation/human/mute/on_losing(mob/living/carbon/human/owner)
+	if(..())	return
+	owner.disabilities &= ~MUTE
+
+/datum/mutation/human/smile
+	name = "Smile"
+	quality = MINOR_NEGATIVE
+	text_indication = "<span class='notice'>You feel so happy. Nothing can be wrong with anything. :)</span>"
+	lose_indication = "<span class='notice'>Everything is terrible again. :(</span>"
+
+/datum/mutation/human/smile/say_mod(var/message)
```

Comment:
don't think so, no

i don't trust my ability to code that without it breaking too